Integrations/FidelisElevateNetwork/FidelisElevateNetwork.py

```diff
@@ -675,6 +677,79 @@ def list_metadata():
     return_outputs(tableToMarkdown('Found {} Metadata:'.format(len(data)), data), context, results)
 
 
+def request_dpath(alert_id):
+
+    res = http_request('GET', '/j/rest/v1/alert/dpath/{}/'.format(alert_id))
+    dpath = res.get('decodingPaths')[0]
+    link_path = dpath.get('linkPath')
+    encoded_path = urllib.quote(link_path)
+    return encoded_path
+
+
+def download_malware_file_request(alert_id):
+
+    dpath = request_dpath(alert_id)
+    query_params = {
+        'uid': SESSION_ID,
+        'alert_id': alert_id,
+        'type': '1',
+        'params': dpath
+    }
+    res = http_request(
+        'GET',
+        '/query/tcpses_getfile.cgi',
+        params=query_params,
+        is_json=False)
+
+    return res
+
+
+def download_malware_file():
+    """
+    Download specific malware from the alert
+    """
+    alert_id = demisto.args().get('alert_id')
+    file_name = request_dpath(alert_id)
+    decoded_file_name = urllib.unquote(file_name)
+    results = download_malware_file_request(alert_id)
+
+    demisto.results(fileResult(
+        decoded_file_name + '.zip',
+        results,
+        file_type=entryTypes['entryInfoFile']))
```

Comment:
should malware file be downloaded as "info file" and not as "file"?
`InfoFile` is for reports and logs, `File` is for incident related files